x-pack/test/ingest_manager_api_integration/apis/epm/file.ts

```diff
@@ -40,7 +40,8 @@ export default function ({ getService }: FtrProviderContext) {
       }
     });
 
-    it('fetches a .json kibana visualization file', async function () {
+    // skipping because the content-type returned by the registry is application/json
```

Comment:
I am not sure to understand but it looks to me that the issue is fixed and we should probably update the test instead no? https://github.com/elastic/package-registry/issues/590

Reply:
Yes, but we can also do it in a follow-up PR and get @jonathan-buttner unblocked on this one. Either way is fine with me.

Reply:
Thanks @skh and @nchaulet I'll switch them over to `application/json` 👍